lab-ron/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lab-ron",
+  "version": "1.0.0",
+  "description": "single-source-api",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Rubiksron/lab-08-single-resource-api.git"
+  },
+  "author": "Ron Dunphy",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Rubiksron/lab-08-single-resource-api/issues"
+  },
+  "homepage": "https://github.com/Rubiksron/lab-08-single-resource-api#readme",
+  "dependencies": {
+    "chai": "^3.5.0",
```

Comment:
Your dependencies are mixed, and you have dev tools in the requirements for the app. need to separate those out.